### ISSUE TITLE
Latest version doesn't support the @nx scope

### ISSUE THREAD

**Issue by austinbiggs** [etkachev/nx-webstorm#108]
## Current Behavior
When using the updated `@nx` scope for npm packages; the console finds no plugins...

Read about the package rescope: https://nx.dev/recipes/other/rescope#rescope-packages-from-@nrwl-to-@nx

## How to Reproduce
1. Install the rescoped package names in your project
2. Open the console UI
3. See that there is "nothing to show"

## Expected Behavior
The generator options should show.

## Screenshots

<img width="402" alt="image" src="https://github.com/etkachev/nx-webstorm/assets/930864/3777613f-779e-4622-9ab4-391c2773b487">

<img width="1013" alt="image" src="https://github.com/etkachev/nx-webstorm/assets/930864/458627fe-bdf1-4cd7-afbf-c2f02b7fe69e">

**Desktop (please complete the following information):**
 - OS: macOS Ventura
 - RubyMine 2023.1.2

**Comment by etkachev** [comment]
Yes you are correct. I'll try to find some time to add additional support for this.
